### ISSUE TITLE
Question about pretraining 

### ISSUE THREAD

**Issue by htw2012** [google-research/bert#171]
Hi, I have some questions about  pre-training as follows:
1. I wanna train my own model from scratch and produce the `vocab.txt` by characters. There are some low-frequency words, should low-frequency words be deleted from the dictionary?
2. If I train my model from Chinese BERT-Base, shall I add the new words from my corpus to the `vocab.txt` that BERT-Base released ? 
3. During the pre-training, under the same conditions, the larger training samples, the better accuracy for MLM and NSP?

Thank you in advance.

**Comment by dvector89** [comment]
1. If you delete some low-frequency words in vocab, you should add a UNK in your vocab.
2. If you change the vocab, you cannot use the released model.

**Comment by artemisart** [comment]
In fact you can reuse the released model if you add words by replacing the unused tokens (but you will have to train them).

**Comment by zhangfazhan** [comment]
@artemisart after i add new words by replacing the unused tokens, how should i train these new words ? thank you!

**Comment by xwzhong** [comment]
@zhangfazhan https://github.com/google-research/bert/issues/155 here is some advice
